wiki/Community/Profile_badge/de.md

```diff
@@ -0,0 +1,41 @@
+---
+tags:
+  - badges
+  - profile badges
+  - user badge
+  - user badges
+---
+
+# Profilabzeichen
+
+*Nicht zu verwechseln mit Gruppenabzeichen, die ebenfalls auf Nutzerprofilen erscheinen.*\
+*Für Regeln für die Verwendung als Turnierpreise, siehe: [Offizielle Turnierunterstützung § Profilabzeichen](/wiki/Tournaments/Official_support#profilabzeichen)*
+
+![Screenshot von den Abzeichen von HappyStick auf der Webseite](img/HappyStick.png "Profilabzeichen werden unter dem Avatar und dem Nutzernamen auf der Webseite angezeigt.")
+
+**Profilabzeichen** (häufig nur *Abzeichen* genannt) sind kleine Grafiken auf dem Nutzerprofil, die für verschiedene Erfolge verliehen werden. Sie werden meist als Preise für [Turniere](/wiki/Tournaments) und [Wettbewerbe](/wiki/Contests) vergeben, haben aber auch andere Verwendungszwecke, wie beispielsweise als Belohnungen für [Community-Mitwirkende](/wiki/People/Community_Contributors), [Beatmap Spotlights](/wiki/Beatmap_Spotlights#rewards) und fortgeführte Gruppenmitgliedschaft.
+
+Wenn man mit dem Mauszeiger über das Abzeichen fährt, wird ein Tooltip mit weiteren Details darüber angezeigt, warum das Abzeichen verliehen wurde.
+
+## Turnierplatzierung
+
+*Hauptseite: [Abzeichen-gewichtete Platzierung](/wiki/Tournaments/Badge-weighted_seeding)*
+
+Im [Spiel-Client](/wiki/Client) und auf der Webseite sind Abzeichen rein dekorativ. Da die Preisauszeichnungen bei [Turnieren](/wiki/Tournaments) jedoch die Leistung der Spieler zeigen, haben einige Turniere [Setzverfahren](https://en.wikipedia.org/wiki/Seed_(sports)) eingeführt, die die Anzahl der Abzeichen der Spieler berücksichtigen, allgemein bekannt als [Abzeichen-gewichtete Platzierung](/wiki/Tournaments/Badge-weighted_seeding) (im Englischen *badge-weighted seeding* oder *BWS*).
+
+## Geschichte
+
+![Screenshot von den Abzeichen von Uan auf der alten Webseite](img/Uan.png "Auf der alten Webseite wurden Profilabzeichen unter dem Namen und dem Benutzertitel gestapelt.")
+
+Anstatt die Abzeichen horizontal in einem Container zu stapeln, der sich über die gesamte Seite erstreckte, wurden sie auf der alten Webseite vertikal zwischen dem Benutzernamen und der Landesflagge des Nutzers angeordnet.
+
+Ein Nebeneffekt dieses Layout-Schemas war, dass die standardmäßige, nicht erweiterbare Höhe von [Benutzerseiten](/wiki/osu!supporter#editierbare-profilsektion) durch das Sammeln weiterer Abzeichen vergrößert werden konnte. Obwohl dies nicht beabsichtigt war, wurde es zu einem Running Gag in der Turnierszene. ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689) veranschaulichte dies in [einem populären Tweet](https://twitter.com/droombs/status/1036050610687074304), in dem er seinen Rekord für die meisten gewonnenen Abzeichen zum Zeitpunkt der Veröffentlichung zeigte.
```

Comment:
```suggestion
Ein Nebeneffekt dieses Layout-Schemas war, dass die standardmäßige, nicht erweiterte Höhe von [Benutzerseiten](/wiki/osu!supporter#editierbare-profilsektion) durch das Sammeln weiterer Abzeichen vergrößert werden konnte. Obwohl dies nicht beabsichtigt war, wurde es zu einem Running Gag in der Turnierszene. ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689) veranschaulichte dies in [einem populären Tweet](https://twitter.com/droombs/status/1036050610687074304), in dem er seinen Rekord für die meisten gewonnenen Abzeichen zum Zeitpunkt der Veröffentlichung zeigte.
```
Could also write `nicht ausgeklappte` as that's a pretty common thing to say I think.

Also fun fact, I noticed that you can't even expand the userpage on the old site anymore for some reason. Definitely used to work when it was the main page though. Is that why you wrote `nicht erweiterbare`? 😄

Reply:
I somehow understood this sentence as in that the height of the box was **not expandable at all** and this bug caused it to expand in just this circumstance.